HandleNHNotificationCallActivity/handler.ts

```diff
@@ -32,9 +32,9 @@ export const ActivityInput = t.interface({
 
 export type ActivityInput = t.TypeOf<typeof ActivityInput>;
 
-const assertNever = (x: never): never => {
-  throw new Error(`Unexpected object: ${toString(x)}`);
-};
+// tslint:disable:no-any
+const createUnexpecterError = (x: any): Error =>
```

Comment:
The whole point of `assertNever` is to have an assignment to a variable of type `never`; that means no other type can be passed to such function except `never`. This is an important mechanism to mark death branches in code (that is: conditions we assert are impossible to happen), and it is at foundation of every [exhaustive check](https://2ality.com/2020/02/typescript-exhaustiveness-checks-via-exceptions.html).

That said, the return type is not important because the functions either is never called or TS fails with wrong arguments error. 

What about runtime? If this functions is executed, it means the variable contains a value which shape differs from the declared type. As this is a problem that can make the whole execution inconsistent, it's safer to throw an exception and let the error to bubble up.